spaceship/lib/spaceship/tunes/device_type.rb

```diff
@@ -1,7 +1,7 @@
 module Spaceship
   module Tunes
     class DeviceType
-      @types = ['iphone4', 'iphone35', 'iphone6', 'iphone6Plus', 'ipad', 'ipadPro', 'watch', 'appleTV', 'desktop']
+        @types = ['iphone4', 'iphone35', 'iphone6', 'iphone6Plus', 'iphone58', 'ipad', 'ipadPro', 'watch', 'appleTV', 'desktop']
```

Comment:
Was the spacing change here intentional?

Reply:
Nope, sorry